mPemba/veryfi-challenge
# Basic receipt analyzer app

## What was changed
- Simple application for uploading and analyzing receipts

## How to test
Start node-server
Start app 
Upload Receipt and click Analyze Receipt
Confirm data shows up as expected 

![Screen Shot 2023-05-11 at 3 36 46 PM](https://github.com/mPemba/veryfi-challenge/assets/8922510/7ceb07e8-5da7-42fe-a200-c71a854e6600)
